staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protobuf/parser.go

```diff
@@ -66,7 +66,7 @@ func rewriteFile(name string, header []byte, rewriteFn func(*token.FileSet, *ast
 	if _, err := f.Write(body); err != nil {
 		return err
 	}
-	return f.Close()
+	return f.Sync()
```

Comment:
this changes the semantics from flushing to OS buffers to flushing straight to disk. That might have performance implications for this functions. It is unclear if we want to do that. The issue you linked I dont think motivates the change

After some brief research it appears `os.Close` there is sufficient unless there is severe OS/HDD failure happening.